source/common/memory/stats.h

```diff
@@ -2,7 +2,20 @@
 
 #include <cstdint>
 
+#include "envoy/config/bootstrap/v3/bootstrap.pb.h"
+#include "envoy/stats/store.h"
+
+#include "source/common/common/thread.h"
+#include "source/common/protobuf/utility.h"
+
 namespace Envoy {
+
+#define MEMORY_ALLOCATOR_MANAGER_STATS(COUNTER) COUNTER(released_by_timer)
```

Comment:
As part of this change i think we should expose stat for [memory release failures](https://github.com/google/tcmalloc/blob/d07e9d9d81c0970218b32099f552aeb70edd4ddd/tcmalloc/global_stats.cc#L639C20-L639C43)

Reply:
I'm not familiar with this, can you provide more info?

Reply:
oh right, forgot about this one. Will do tomorrow!

Reply:
gperf tcmalloc does not seem to export such stat unfortunately, only tcmalloc: https://github.com/google/tcmalloc/blob/master/tcmalloc/system-alloc.h#L55
So let's not expose such stat